econplayground/templates/main/graph_detail.html

```diff
@@ -30,11 +30,6 @@
     </div>
 
     {% if i_am_instructor %}
-        <a role="button" class="btn btn-danger btn-sm float-md-right"
```

Comment:
@nikolas I moved this to React like we discussed, but would taking it out of this conditional in Django permit non instructors to see the button?

Reply:
Removing this conditional would allow non-instructors to see anything that's within this conditional.. i'm not sure i understand the question. Because you've moved it to the react side, the visibility will be handled differently. e.g., you've put it in GraphEditor, which currently never gets displayed to non-instructors.